pkg/receiver.go

```diff
@@ -100,6 +111,35 @@ func (w *WebRTCReceiver) OnCloseHandler(fn func()) {
 	w.onCloseHandler = fn
 }
 
+func (w *WebRTCReceiver) AddSender(pid string, sender Sender) {
+	w.Lock()
+	defer w.Unlock()
+	w.senders[pid] = sender
+}
+
+func (w *WebRTCReceiver) DeleteSender(pid string) {
+	w.Lock()
+	defer w.Unlock()
+	delete(w.senders, pid)
+}
+
+func (w *WebRTCReceiver) SetSimulcast(ssrc uint32) {
+	w.simulcast = true
```

Comment:
this is set explicitly here but also derived from track.RID() above which is a bit confusing